src/instructlab/eval/mt_bench_common.py

```diff
@@ -270,17 +273,16 @@ def chat_completion_openai(
                 temperature=temperature,
                 max_tokens=max_tokens,
             )
-            output = response.choices[0].message.content
-            break
-        except openai.OpenAIError as e:
-            if i == API_MAX_RETRY - 1:
-                # Print error on last try
-                print(type(e), e)
-            else:
-                logger.debug(e)
-            time.sleep(API_RETRY_SLEEP)
-
-    return output
+            return response.choices[0].message.content
```

Comment:
won't this return on the first iteration? Please correct me if I'm missing something

Reply:
Yes. This is what the code does right now, no? It `break`s our of the `for` loop, which makes it immediately `return output`. I think that's by design - the function is banging at the API endpoint until a successful reply comes back, then returns the reply. Only in the unfortunate cases of exceptions we - sometimes - want to retry again.